### PR TITLE
Website blog

### PR DESCRIPTION
**Pull request by Kryan92**

### CODE REVIEW

File: _posts/2017-07-07-tech-behind-dta-website.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+- allow previewing of changes on the site before they go live
+- handle less than 1000 pages
+
+We’ve found that a simple static site meets our needs for the moment and is cheap, fast, secure, and easy to work with.
+
+WARNING: it’s about to get technical 
+
+## The tech behind the site 
+
+The DTA website is built with [Jekyll](https://jekyllrb.com/) — an open source static website generator. The site content is mostly written in [markdown](https://en.wikipedia.org/wiki/Markdown). Jekyll only runs whenever the content changes, and it generates a set of HTML, CSS and JavaScript files. These files are easily hosted on a basic web server.
```
Comment (gordcorp) [inline _posts/2017-07-07-tech-behind-dta-website.md]: The capitalisation of markdown should be consistent - further down we say Markdown, and link somewhere else.

File: _posts/2017-07-07-tech-behind-dta-website.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+thumbnail: /images/blog-thumbnails/Blog-Thumb-DTA-website.png
+hero-image: /images/blog-content/Blog-Hero-DTA-website.png
+searchexcerpt: "The DTA website is an important part of our online presence. But as a small agency we have found our needs can be met with a simple static site. In this blog post we share our approach to development and the technology behind the DTA website."
+---
+
+![An illustration of a pony with a rainbow coloured mane.]({{ site.url }}{{ site.baseurl }}{{ page.hero-image }})
+
+*Caption: 'I want a pony' – a demonstration of a want, not a need.*
+
+The technical needs for our [website](https://www.dta.gov.au/) are relatively simple. We took a minimum viable product (MVP) approach to development – creating a simple static site – with a view that we would iterate and evolve the website as the DTA grows.
```
Comment (gordcorp) [inline _posts/2017-07-07-tech-behind-dta-website.md]: Should be em-dashes i think?

File: _posts/2017-07-07-tech-behind-dta-website.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+Our site needs to be able to:
+- allow publishing of simple, informative content
+- support less than ten page templates
+- allow previewing of changes on the site before they go live
+- handle less than 1000 pages
+
+We’ve found that a simple static site meets our needs for the moment and is cheap, fast, secure, and easy to work with.
+
+WARNING: it’s about to get technical
```
Comment (gordcorp) [inline _posts/2017-07-07-tech-behind-dta-website.md]: full stop?

File: _posts/2017-07-07-tech-behind-dta-website.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+We have found this simple tech approach to be very cheap compared to the alternatives, but is still powerful enough to suit our publishing needs.
+
+## Using the UI-Kit 
+
+The DTA website is styled using the [UI-Kit](https://github.com/AusDTO/gov-au-ui-kit) and the [DTA Design Guide](http://guides.service.gov.au/design-guide/). 
+
+The UI-Kit is a whole-of-government toolkit being created by the DTA to provide government with a set of accessible web components to use on their websites. Agencies can use it instead of creating custom designs and styles for their sites — the intention is to create more consistency across government sites. It is designed to meet high accessibility standards required for the [Digital Service Standard](https://www.dta.gov.au/standard/). 
+
+We still sometimes need to write our own CSS, but the UI-Kit gives us an easy to use starting point
```
Comment (gordcorp) [inline _posts/2017-07-07-tech-behind-dta-website.md]: add full stop

File: _posts/2017-07-07-tech-behind-dta-website.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+- Changes are deployed to the staging site for sign-off.
+- Changes are then merged with the ‘master’ branch which triggers a deployment to the live site. 
+
+Each part of the deployment happens automatically whenever the ‘develop’ or ‘master’ branch is updated. This speeds up and simplifies the release process for content publishers. The repository contents are the single source of truth, and the staging site is an exact copy of what will go to the live site.
+
+Our website is a Commonwealth record and needs to be managed in accordance with the Archives Act 1983 and the [National Archives of Australia guidance](http://www.naa.gov.au/Images/archweb_guide_tcm16-47165.pdf). Since every change to the site is tracked in version control, we can easily meet this requirement by going back and rebuilding the site as it was at any point in time.
+
+## Hosting our site — cloud.gov.au
+
+The DTA website is hosted on [cloud.gov.au](https://www.dta.gov.au/what-we-do/platforms/cloud/) – the platform the DTA is developing to make it easier for government to build digital services. Cloud.gov.au saves time and effort by managing infrastructure, so a team can focus on building the service.
```
Comment (gordcorp) [inline _posts/2017-07-07-tech-behind-dta-website.md]: em-dash?

File: _posts/2017-07-07-tech-behind-dta-website.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+Cloud.gov.au has an API which allows deployments to be entirely automated. We specify the [static build pack](http://docs.cloudfoundry.org/buildpacks/staticfile/index.html) in our [manifest file](https://github.com/AusDTO/dta-website/blob/develop/manifest-production.yml), and the platform takes care of provisioning a server and inserting the built website files. This means our exposure to security attacks is minimised, and we do not have to spend time regularly applying platform updates.
+
+We use [Cloudfront](https://github.com/AusDTO/dta-website/blob/develop/manifest-production.yml) as our content delivery network, and requests are routed to a HTTPS termination layer on cloud.gov.au before being forwarded to our web server.
```
Comment (gordcorp) [inline _posts/2017-07-07-tech-behind-dta-website.md]: Hmm this link doesnt make sense. Change to https://aws.amazon.com/cloudfront/

File: _posts/2017-07-07-tech-behind-dta-website.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+## External services and plugins
+
+A static site on its own has limitations, but this is where external services are useful. The ones we currently use are:
+- [Disqus](https://disqus.com/) for our blog commenting system
+- [Youtube](https://www.youtube.com/channel/UCmDkFN3UlK2wSKDQQhd-Y-A) for hosting our videos
+- [Recruiterbox](https://recruiterbox.com/) for showing [our open jobs](https://www.dta.gov.au/who-we-are/corporate/jobs/)
+- [Google Apps Script](https://www.google.com/script/start/) for receiving form submissions
+
+Site search is something you might expect to be an external service, but we use [client-side JavaScript](https://www.google.com/script/start/). We could improve our search with an external service, but for our current usage this is free and fast.
```
Comment (gordcorp) [inline _posts/2017-07-07-tech-behind-dta-website.md]: This link should be to https://github.com/slashdotdash/jekyll-lunr-js-search - this is the jekyll plugin we use which creates the search javascript and index at build-time (Well actually we use a slightly modified local version, but let's not complicate things!)

Comment (Kryan92) [inline _posts/2017-07-07-tech-behind-dta-website.md]: Do you mean the link to 'client-side JavaScript'?

File: _posts/2017-07-07-tech-behind-dta-website.md (hunk 6 above)
Comment (Kryan92) [inline _posts/2017-07-07-tech-behind-dta-website.md]: Done

File: _posts/2017-07-07-tech-behind-dta-website.md (hunk 7 above)
Comment (gordcorp) [inline _posts/2017-07-07-tech-behind-dta-website.md]: Yep

**Pull request merged**